### ISSUE TITLE
`stats` command broken

### ISSUE THREAD

**Issue by ThatOneCalculator** [esmBot/esmBot#143]
### Command that caused the error

stats

### Image that caused the error

none

### Steps to reproduce

`@esmbot stats` produces 
```
Message: TypeError: Cannot read property 'username' of undefined

Stack Trace: TypeError: Cannot read property 'username' of undefined
    at StatsCommand.run (/home/esmBot/esmBot/commands/general/stats.js:19:63)
    at module.exports (/home/esmBot/esmBot/events/messageCreate.js:90:39)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

### Self-hosted instance?

No

### Error file

```
Message: TypeError: Cannot read property 'username' of undefined

Stack Trace: TypeError: Cannot read property 'username' of undefined
    at StatsCommand.run (/home/esmBot/esmBot/commands/general/stats.js:19:63)
    at module.exports (/home/esmBot/esmBot/events/messageCreate.js:90:39)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by ThatOneCalculator** [comment]
![Screenshot_20210728-160225.jpg](https://user-images.githubusercontent.com/44733677/127336119-8d1ab8bd-a6f6-4367-8eaa-ec8804b242de.jpg)

**Comment by TheEssem** [comment]
Seems to be fixed on master with 0af6533.
